pkgs/data/themes/fluent-gtk-theme/default.nix

```diff
@@ -23,13 +23,13 @@ lib.checkListOfEnum "${pname}: tweaks" [ "solid" "float" "round" "blur" "noborde
 
 stdenvNoCC.mkDerivation rec {
```

Comment:
Can you remove the recursion and move to `finalAttrs`?

Reply:
Done.